hello-helm/k8s/values.yaml

```diff
@@ -0,0 +1,9 @@
+environment: demo
+container:
+  name: cloudnatived
+  port: 8080
+  repo: cloudnatived
+  tag: latest
+greeting: Hello
```

Comment:
By the way, I think it makes sense for us to leave config and secrets until the config and secrets chapter. That means we'll be doing `hello`, `hello-k8s`, `hello-helm`, and `hello-helmfile` in this chapter, with no configs or secrets, and then in the next chapter add the config and secrets.